lib/active_record_shards/model.rb

```diff
@@ -9,13 +9,14 @@ def not_sharded
     end
 
     def is_sharded? # rubocop:disable Style/PredicateName
+      sharded_ivar = defined?(@sharded) ? @sharded : nil
```

Comment:
I put the suffix since there's a method called `not_sharded`. Wanted to make sure this code doesn't start shadowing a method, could become confusing.